scripts/build_lbann_lc.sh

```diff
@@ -442,6 +449,16 @@ fi
 CXX_FLAGS="${CXX_FLAGS} -ldl"
 C_FLAGS="${CXX_FLAGS}"
 
+# Hacks to build with NVSHMEM
+if [ ${WITH_NVSHMEM} -ne 0 ]; then
+    if [ "${CLUSTER}" == "lassen" ]; then
+        NVSHMEM_DIR=/usr/workspace/wsb/brain/nvshmem/nvshmem_0.3.3/cuda-10.1_ppc64le
+        CUDA_FLAGS="-gencode=arch=compute_70,code=sm_70"
```

Comment:
Do you need this in addition to the `CMAKE_CUDA_FLAGS` setting in FindNVSHMEM?

Reply:
Inside `FindNVSHMEM.cmake` we set `-gencode=arch=compute_70,code=compute_70`, which generates future-compatible PTX. By also setting `-gencode=arch=compute_70,code=sm_70`, we make sure to compile a cubin specifically for Voltas. I think this is a bit more generalizable for post-Volta GPUs, although it might also be overkill.